app/controllers/two_factor_authentication/otp_verification_controller.rb

```diff
@@ -35,6 +35,10 @@ def confirm_two_factor_enabled
       redirect_to phone_setup_url
     end
 
+    def confirming_phone?
```

Comment:
Shouldn't you be able to check `user_session[:context]` here? I think that should be set to `idv`.

Reply:
Yeah, I think that is set here: https://github.com/18F/identity-idp/blob/master/app/controllers/concerns/phone_confirmation.rb#L4

That will likely be more reliable than `user_session[:jurisdiction]`.

Reply:
It's up to you.  I wanted something as close as possible to the step before the phone verification.

Reply:
The is no user_session[:jurisdiction].  It's user_session[:idv_jurisdiction].  I can change it to :idv if you prefer.

Reply:
Sorry, I mistyped there. I meant to say, "I think `user_session[:context]` is what we want".

Reply:
Yep.  I'll just add an idv_context?  Sound good?

Reply:
The reality is confirmation_context should have been set in that idv code but idv context overwrote it which is why this was broken and never worked.

Reply:
The whole singular context thing is a little flawed because this screen actually exists in multiple contexts.